modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectDefinitionLocalServiceTest.java

```diff
@@ -1302,6 +1333,26 @@ private void _assertSystemObjectFields(
 			expectedObjectField.isState(), objectField.isState());
 	}
 
+	private void _assertSystemObjectTitleField(
+			String objectDefinitionName, String objectFieldName)
+		throws Exception {
+
+		SystemObjectDefinitionMetadata systemObjectDefinitionMetadata =
+			_systemObjectDefinitionMetadataRegistry.
+				getSystemObjectDefinitionMetadata(objectDefinitionName);
+
+		ObjectDefinition systemObjectDefinition =
+			_objectDefinitionLocalService.fetchSystemObjectDefinition(
+				systemObjectDefinitionMetadata.getName());
```

Comment:
@p-albuquerque do we really need to call the `getSystemObjectDefinitionMetadata`? Can't we `fetchSystemObjectDefinition` using the `objectDefinitionName`?